.github/workflows/config.yml

```diff
@@ -45,13 +49,14 @@ jobs:
 
           if [ ! -e "venv" ]; then
             echo "installing venv"
-            NO_HOOKS=1 SETUP_PY2=no .hooks/bootstrap
```

Comment:
There are two different ".hooks/" folders and the workflow calls the remote script in demisto/content/.hooks/bootstrap not the local demisto/content-ci-cd-template/.hooks/bootstrap it used to work just fine up until demisto/content's switch to poetry. Copied the remote .hooks folder to local and modified the workflow to use the local .hooks folder. Also, the workflow uses poetry now.